front/app/components/admin/PostManager/components/PostPreview/Initiative/AdminInitiativeEdit.tsx

```diff
@@ -83,16 +81,20 @@ const AdminInitiativeEdit = ({ goBack, initiativeId }: Props) => {
   }
 
   return (
-    <Container>
+    <Box
+      minHeight="100%"
+      width="100%"
+      position="relative"
+      flexDirection="column"
+      alignItems="center"
+      bgColor="white"
+      border="1px solid white"
+      borderRadius={theme.borderRadius}
+    >
```

Comment:
Does the same as the imported Container component did, but needed to add 2 custom styles.